packages/bcer-retailer-app/app/src/hooks/useCsvValidator.ts

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import * as yup from 'yup';
+
+export const useCsvValidator = () => {
+    const [errors, setErrors] = useState<Array<{row: string, field: string, message: string}>>()

```

Comment:
Looks like there's a bit extra spacing here compared to the other files. Tabs -> 2 spaces?